### PR TITLE
Fixed the CV issues from the Workshop on Set-1 for version-2

### PR DESCRIPTION
**Pull request by arunherb**
Workshop for set-1 controlled vocabularies have been updated.

### CODE REVIEW

**Comment by junrongYu** [comment]
Hi @arunherb , for the `land-use history` update, can you please remove the attribute concept file & move the LUT URI to metadata file, update the value type in metadata to `tern:IRI`? Can you please also state the changes in commit message?

File: vocab_files/observable_properties_by_module/plot-description/plot-description-enhanced-protocol/land-use-history.ttl [hunk 1]
```diff
@@ -7,10 +7,11 @@ PREFIX xsd: <http://www.w3.org/2001/XMLSchema#>
 <https://linked.data.gov.au/def/nrm/68497b35-8d91-46dc-8829-9626f1cf0178>
     a urnc:ObservablePropertyMeta ;
     schema:url "https://github.com/ternaustralia/dawe-rlp-vocabs/tree/main/vocab_files/observable_properties_by_module/plot-description/plot-description-enhanced-protocol/land-use-history.ttl"^^xsd:anyURI ;
+    tern:hasCategoricalCollection <http://www.neii.gov.au/def/voc/ACLUMP/australian-land-use-and-management-classification/Australian-Land-Use-and-Management-Classification> ;
```
Comment (junrongYu) [inline vocab_files/observable_properties_by_module/plot-description/plot-description-enhanced-protocol/land-use-history.ttl:10]:
```suggestion
    urnp:categoricalValuesCollection <http://www.neii.gov.au/def/voc/ACLUMP/australian-land-use-and-management-classification/Australian-Land-Use-and-Management-Classification> ;
```

**Review by junrongYu — CHANGES_REQUESTED** [review]

**Review by junrongYu — APPROVED** [review]

**Comment by junrongYu** [comment]
Hi @arunherb , for this update `Remove OP species identification status and its metadata files in ` floristics - standard``, can you please bring the concept file back cuz it may be used in set2/3, and remove its URI form the OPs collection file of `floristics - standard`. Thanks!

**Comment by junrongYu** [comment]
Hi @arunherb , same here, for `Remove OPs field species name and growth form and their metadata files in cover-standard`, can you please remove the URIs of OPs `field species name` and `growth form` from the OP collection file in `cover - standard`. Thanks!

**Comment by junrongYu** [comment]
Hi @arunherb , for this update 'Rename OP fractional cover responses to fractional cover types, in concept file, metadata file;', can you please remove the extra `v` in file url, it's `vvocabs_file`. Thanks!

**Comment by arunherb** [comment]
> Hi @arunherb , for this update 'Rename OP fractional cover responses to fractional cover types, in concept file, metadata file;', can you please remove the extra `v` in file url, it's `vvocabs_file`. Thanks!

Done

**Comment by arunherb** [comment]
> Hi @arunherb , same here, for `Remove OPs field species name and growth form and their metadata files in cover-standard`, can you please remove the URIs of OPs `field species name` and `growth form` from the OP collection file in `cover - standard`. Thanks!

Done

**Comment by junrongYu** [comment]
Hi @arunherb , for `floristics - standard`, you've added the metadata URI of `dead branch` to the collection file, should be the concept URI, can you please correct it? Thanks!

**Comment by arunherb** [comment]
> Hi @arunherb , for `floristics - standard`, you've added the metadata URI of `dead branch` to the collection file, should be the concept URI, can you please correct it? Thanks!

This is probably 'cover - standard'; It has been updated. Thanks

**Comment by junrongYu** [comment]
Hi @arunherb , thanks for the update, can you please fix these:
1. for the 'Remove attribute land-use history;Add OP land-use history and its metadata file, feature type is land surface', feature type is `land surface`;
2. for the 'Remove attribute source of identification and its relationship with OP field species name', can you please bring the concept file back, it's used un set2;
3. for the metadata file of OP `animal count`, should the feature type be `animal occurrence`?
4. for both `animal count` and `plant count`, can you please add those attributes linked with `number of individuals` with those 2 new OPs in metadata? they are observation method tier1/2/3, and observation notes
Thanks!

**Comment by arunherb** [comment]
> Hi @arunherb , thanks for the update, can you please fix these:
> 
> 1. for the 'Remove attribute land-use history;Add OP land-use history and its metadata file, feature type is land surface', feature type is `land surface`; **(DONE)**
> 2. for the 'Remove attribute source of identification and its relationship with OP field species name', can you please bring the concept file back, it's used un set2; **(DONE)**
> 3. for the metadata file of OP `animal count`, should the feature type be `animal occurrence`?**(DONE)**
> 4. for both `animal count` and `plant count`, can you please add those attributes linked with `number of individuals` with those 2 new OPs in metadata? they are observation method tier1/2/3, and observation notes - I think we decided not to have this concept 'number of individuals', instead I have renamed this to 'indivdiual count' and linked it to animal and plant count as a broader concept. _For the second part 'observation method tier 1/2/3' I need more clarity as to what needs to be added where._
Thanks!

**Comment by junrongYu** [comment]
> > Hi @arunherb , thanks for the update, can you please fix these:
> > 
> > 1. for the 'Remove attribute land-use history;Add OP land-use history and its metadata file, feature type is land surface', feature type is `land surface`; **(DONE)**
> > 2. for the 'Remove attribute source of identification and its relationship with OP field species name', can you please bring the concept file back, it's used un set2; **(DONE)**
> > 3. for the metadata file of OP `animal count`, should the feature type be `animal occurrence`?**(DONE)**
> > 4. for both `animal count` and `plant count`, can you please add those attributes linked with `number of individuals` with those 2 new OPs in metadata? they are observation method tier1/2/3, and observation notes - I think we decided not to have this concept 'number of individuals', instead I have renamed this to 'indivdiual count' and linked it to animal and plant count as a broader concept. _For the second part 'observation method tier 1/2/3' I need more clarity as to what needs to be added where._
> >    Thanks!

Hi @arunherb , yes we removed the `number of individuals`, replaced it with `animal count` / `plant count`, this is definitely correct. However, when removed its metadata file, we lost the relationship between this OP and its attributes. Attributes `observation method tier 1/2/3` and `observation notes` are attributes of all OPs in `opportune` module.

**Review by junrongYu — APPROVED** [review]

**Pull request merged**